sdcm/tester.py

```diff
@@ -856,7 +857,28 @@ def setUp(self):  # pylint: disable=too-many-branches,too-many-statements
                 db_cluster.validate_seeds_on_all_nodes()
                 validate_raft_on_nodes(nodes=db_cluster.nodes)
 
+    def _switch_to_network_replication_strategy(self, keyspaces: List[str]) -> None:
+        """Switches replication strategy to NetworkTopology for given keyspaces.
+        """
+        node = self.db_cluster.nodes[0]
+        nodes_by_region = self.db_cluster.nodes_by_region()
+        region = list(nodes_by_region.keys())[0]
+        dc_name = self.db_cluster.get_nodetool_info(nodes_by_region[region][0])['Data Center']
+        for keyspace in keyspaces:
+            replication_strategy = ReplicationStrategy.get(node, keyspace)
+            if not isinstance(replication_strategy, SimpleReplicationStrategy):
+                # no need to switch as already is NetworkTopology
+                continue
+            self.log.info("Switching replication strategy to Network for '%s' keyspace", keyspace)
+            if keyspace == "system_auth" and replication_strategy.replication_factor != len(nodes_by_region[region]):
+                self.log.warning("system_auth keyspace is not replicated on all nodes (%s)",
+                                 replication_strategy.replication_factor / len(nodes_by_region[region]))
+            network_replication = NetworkTopologyReplicationStrategy(
+                **{dc_name: replication_strategy.replication_factor})
+            network_replication.apply(node, keyspace)
+
```

Comment:
Let's run "Desc keyspace" for the changed keyspace, so we will have the output in the log.

Reply:
Added appropriate code.